### PR TITLE
Added support for Fedora

### PR DESCRIPTION
**Pull request by marcellussiegburg**
These changes add **support for more rpmfusion urls and for Fedora** systems. The default behaviour is not changed.

However it should be considered to release this software under a special **license**. I would agree to any license model for my contributions, if GPL, Apache or MIT license would not matter to me.

### CODE REVIEW

**Comment by lboynton** [comment]
Thanks! However I'm getting an error here:

`'undef' from left operand of 'in' expression is not a string at /srv/puppet/modules/rpmfusion/manifests/params.pp:58`

**Comment by marcellussiegburg** [comment]
I am not able to reproduce the error, can you provide some of your facter facts (e.g os_maj_version should be provided by able; maybe try $::operatingsystemmajrelease instead of $::os_maj_version).

File: manifests/params.pp [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+          $version = $::operatingsystemrelease
+        } else {
+          $version = ''
+        }
+      }
+      default: {
+        $supported_gpg = ['',6]
+        $type = 'el'
+        if $rpmfusion::with_version == true {
+          $version = $::os_may_version
```
Comment (lboynton) [inline manifests/params.pp:48]: Typo here, should be $::os_maj_version

Comment (marcellussiegburg) [inline manifests/params.pp:48]: ## Thanks, this is exactly the line where I guessed the error to be. I will fix it until tomorrow.

Diese Nachricht wurde von meinem Mobiltelefon gesendet.

Comment (lboynton) [inline manifests/params.pp:48]: Since it's just a small fix, I can do it and get this merged in.

**Comment by lboynton** [comment]
I've merged your changes. I will probably go for Apache for the license.

**Comment by marcellussiegburg** [comment]
Thanks for merging. I would agree with Apache for my contributions as well.

**Pull request closed without merging**